tests/Detector.Tests/Hugo/HugoDetectorTest.cs

```diff
@@ -78,8 +79,8 @@ public void IsHugoApp_ReturnsTrue_ForAllSupportedConfigurationKeys_InTomlFile(st
         }
 
         [Theory]
-        [InlineData(HugoConstants.JsonFileName)]
-        [InlineData(HugoConstants.ConfigFolderName, HugoConstants.JsonFileName)]
+        [InlineData("config.json")]
```

Comment:
Just curious: we can't directly use `HugoConstants.JsonFileNames.First()` in these InlineData fields to remain consistent?